commitizen/config/__init__.py

```diff
@@ -39,7 +46,9 @@ def read_cfg() -> BaseConfig:
         elif "yaml" in filename.suffix:
             _conf = YAMLConfig(data=data, path=filename)
 
-        if _conf.is_empty_config:
+        if filepath is not None and _conf.is_empty_config:
+            raise ConfigFileIsEmpty()
```

Comment:
I'm not sure whether we really need to raise an error here. @woile @noirbizarre what do you think?

Reply:
IMO, if the file path is been given it must be have some content inside, otherwise there's no need to specify the config file path.

However if the empty config always acceptable, I'll remove the line as well.

Reply:
After a second thought, I think the current implementation is better. If the user want to ignore it, they can always use no-raise